src/main/java/ui/workout/workoutsession/WorkoutSessionUi.java

```diff
@@ -12,18 +12,20 @@ public static void printError() {
     }
 
     public static void addFormatError() {
-        printer.showToUser("Wrong format, please enter in the format: \n\t "
+        printer.showToUser("Wrong format, please enter in the format:\n\t "
```

Comment:
Since WorkoutSessionUi extends CommonUi already, should the line "private static CommonUi printer = new CommonUi();" be removed? and use the "showToUser" method of CommonUi?

Reply:
My calls are all static tho, so printer has to be static

Reply:
You can make use of "showUser" method in CommonUi. It's also static.

Reply:
Oh I didnt see that, thanks